pkg/build/registry/buildconfiginstantiate/rest.go

```diff
@@ -260,25 +258,24 @@ func (h *binaryInstantiateHandler) handle(r io.Reader) (runtime.Object, error) {
 		Stderr:    true,
 		Container: buildstrategy.GitCloneContainer,
 	}
-	location, transport, err := pod.AttachLocation(h.r.PodGetter, h.r.ConnectionInfo, h.ctx, buildPodName, opts)
+	location, _ /*transport*/, err := pod.AttachLocation(h.r.PodGetter, h.r.ConnectionInfo, h.ctx, buildPodName, opts)
 	if err != nil {
 		if errors.IsNotFound(err) {
 			return nil, errors.NewNotFound(kapi.Resource("pod"), buildPodName)
 		}
 		return nil, errors.NewBadRequest(err.Error())
 	}
-	tlsClientConfig, err := knet.TLSClientConfig(transport)
-	if err != nil {
-		return nil, errors.NewInternalError(fmt.Errorf("unable to connect to node, could not retrieve TLS client config: %v", err))
-	}
-	upgrader := spdy.NewRoundTripper(tlsClientConfig, false)
-	exec, err := remotecommand.NewStreamExecutor(upgrader, nil, "POST", location)
+	// tlsClientConfig, err := knet.TLSClientConfig(transport)
+	// if err != nil {
+	// 	return nil, errors.NewInternalError(fmt.Errorf("unable to connect to node, could not retrieve TLS client config: %v", err))
+	// }
+	// TODO: fixme
+	exec, err := remotecommand.NewSPDYExecutor(&restclient.Config{}, "POST", location)
 	if err != nil {
 		return nil, errors.NewInternalError(fmt.Errorf("unable to connect to server: %v", err))
 	}
 	streamOptions := remotecommand.StreamOptions{
-		SupportedProtocols: kubeletremotecommand.SupportedStreamingProtocols,
```

Comment:
surprising

Reply:
That's still broken, that's one of the pieces I need to revisit.

Reply:
@soltysh is there a follow up issue or note somewhere so we don't forget?